braga/README.md

```diff
@@ -0,0 +1,655 @@
+
+The **rent** for **premises** in London
+is the _**tip** of the **cost iceberg**_! <br />
+By _far_ the biggest "_space_" cost is the ***cumulative rent***
+all the people _attending_ <br />
+Founders & Coders (_and subsequently working for @dwyl_)
+pay to _live_ in the city.
+
++ The _average **rent**_ for a _room_ in London
+is **£630/month** (_including bills_).
++ For those who use _public transport_
+a **travel pass** costs a ***minimum*** of
+[**£120/month**](http://content.tfl.gov.uk/adult-fares-2017.pdf) <br />
++ If people eat _frugally_ by preparing meals in bulk/advance,
+_limit alcohol intake_ and "_eating out_"
+they can survive on **£4/day** for **food** so ***£120/month***.
+
+> The _**bare minimum**_ total living cost in London is ***£870/month***. <br />
+This does not factor the wasted _time_ people have commuting <br />
+to-from Mile End each day (_average 1.5h per day!_)
+
+> TODO: @iteles _please_ Insert **Cost of Living Questionnaire** Data/Summary `here` ... https://github.com/dwyl/phase-two/issues/57
+
+## What?
+
+`Braga House` is a **hub** for creative technology learning,
+healthy and sustainable living.
+
+Our `mission` is to give people a comfortable
+yet **_highly_ cost-effective** place to live <br />
+while they are learning the skills to work in (_or start_)
+the top creative technology companies.
+
+
+### Saving _Everyone_ Money/Time
+
+Setting up `Braga House` will cut down on
+_London_ living expenses by ***Two Thirds*** <br />
+and give _everyone_ ***more time*** to learn, create and enjoy life!
+
+The ***Total Cost*** of living in the `Braga House`
+will be _**€256 per Month**_ (_per person_).
+
+This price is _exceptionally good value-for-money_
+based on our (_extensive_) market research
+and considering all the benefits people will get (_see below for "what's included"_).
+
+#### _Financially Sustainable_ But _Not For Profit_
+
+At €256/person/month the `Braga House` will cover all costs
+and make a _**small surplus**_ which will be
+***re-invested*** into the house & community.
+
+#### _What's Included_?
+
++ A **single bed**<sup>1</sup> in a ***shared room***<sup>2</sup> with:
+  + an orthopedic memory foam mattress (_or alternative of your choice up to €200_)
+  + your **choice** of **pillow** (_because pillows are a v. personal thing and make **all** the difference to sleep quality!_)
+  + **bed linen** (_highest quality_) is **changed weekly** by housekeeper.
+  (_if you prefer a higher frequency inform the "house parent"_)
++ **State-of the Art Kitchen** with every appliance/gadget, utensil and utility
+you could _ever_ need to unleash your ***culinary creativity** if choose to.
+  + **3 Large Fridges** so everyone has space for their food.
+  + **3 Dishwashers** so nobody has to wash crockery!
++ ***Unlimited*** Healthy Vegan Soup (_freshly made each day_)
++ ***Unlimited*** Fresh Fruit & Vegetables (_personal use only, please don't start a restaurant with the House's "free" ingredients!_)
```

Comment:
_How_ could smoothies cost us €10/day ?
In the UK my smoothies cost me £2/day using Retail ingredients.
We will be buying Wholesale ingredients which are usually _half the price_ and _most_ people will NOT be drinking 2Litres of Smoothie per day. 
But...
Please feel free to change this to anything it needs to be.
Thanks! ✨